### PR TITLE
Add Heisenbridge to IRC bridges

### PR DESCRIPTION
**Pull request by hifi**
:airplane: Whoosh, a challenger appears! :partying_face: 

I looked up other projects along the template to get it set up like the other bridges were so it shows up below matrix-appservice-irc. I don't know if it's allowed to add the featured flag so advice if I need to revise this.

Thanks.

### CODE REVIEW

**Comment by hifi** [comment]
Ping @SpiritCroc and @benparsons 

There's also a few other project PRs before this that need to be looked at.

File: gatsby/content/projects/bridges/heisenbridge.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+license: MIT
+repo: https://github.com/hifi/heisenbridge
+room: "#heisenbridge:vi.fi"
+thumbnail: /images/irc.png
+maturity: Alpha
+featured: true
+bridges: IRC
+---
+
+A bouncer style Matrix IRC bridge with focus on simplicity for personal use on your own homeserver. No complex configuration files or databases required.
```
Comment (jaller94) [inline gatsby/content/projects/bridges/heisenbridge.mdx:19]: [Compound adjectives](https://www.thefreedictionary.com/Compound-Adjectives.htm) use one or more hyphens as delimiters.

This is just a comment, not a review. I'm not a repo owner.

```suggestion
description: a bouncer-style Matrix IRC bridge
author: hifi
home: https://github.com/hifi/heisenbridge
language: Python
license: MIT
repo: https://github.com/hifi/heisenbridge
room: "#heisenbridge:vi.fi"
thumbnail: /images/irc.png
maturity: Alpha
featured: true
bridges: IRC
---

A bouncer-style Matrix IRC bridge with focus on simplicity for personal use on your own homeserver. No complex configuration files or databases required.
```

Comment (hifi) [reply gatsby/content/projects/bridges/heisenbridge.mdx]: Thank you for the heads up. I'm not a native speaker and this things slip through. I rebased and updated with your suggestion.

**Comment by MadLittleMods** [comment]
Tested locally, looks good 👍 

http://localhost:8000/bridges/#irc
Screen Shot 2021-05-25 at 5 27 48 PM

http://localhost:8000/docs/projects/bridge/heisenbridge
Screen Shot 2021-05-25 at 5 28 21 PM

**Pull request merged**